docs/dev-guide/deployments_table_migration.md

```diff
@@ -0,0 +1,85 @@
+# Deployments Table Migration
```

Comment:
Per [this comment](https://github.com/caraml-dev/merlin/pull/483#pullrequestreview-1713955770), can we just turn this into up / down sql migration files? This will make it easier to propagate this to all envs.

Is there any concern with capturing this data migration in the `db-migrations` scripts?

Reply:
Oh, we can. Updated here: https://github.com/caraml-dev/merlin/pull/483/files#diff-39f6fa7711419b318e134517b1068e6ffb81268adc7f36fa0d215a58af5ea697

Btw, the down.sql file is empty though. Because it's backward compatible after we update the deployment status. Do you have any suggestions?

Reply:
The script looks good to me! :)